src/unreal_plugin/Source/UnrealDeadlineCloudService/Private/Tests/Integration/DeadlinePluginTest_CreateJobTest.cpp

```
@@ -118,6 +141,167 @@ class RestoreQueueCommand : public IAutomationLatentCommand
+                if (KeyValuePairs[i].Split(TEXT("="), &Key, &Value))
+                {
+                    UE_LOG(LogCreateJobTest, Display, TEXT("Split into key='%s', value='%s'"), *Key, *Value);
+
+                    if (Key == TEXT("farm_id") && !Value.IsEmpty())
+                    {
+                        // Check if the farm value is actually changing
+                        if (Settings->WorkStationConfiguration.Profile.DefaultFarm != Value)
+                        {
+                            UE_LOG(LogCreateJobTest, Display, TEXT("Setting farm to '%s'"), *Value);
+                            Settings->WorkStationConfiguration.Profile.DefaultFarm = Value;
+                            farmChanged = true;
+                        }
+                        else
+                        {
+                            UE_LOG(LogCreateJobTest, Display, TEXT("Farm value unchanged (already '%s'), skipping update"), *Value);
+                        }
+                    }
+                    else if (Key == TEXT("queue_id") && !Value.IsEmpty())
+                    {
+                        // Check if the queue value is actually changing
+                        if (Settings->WorkStationConfiguration.Farm.DefaultQueue != Value)
+                        {
+                            UE_LOG(LogCreateJobTest, Display, TEXT("Setting queue to '%s'"), *Value);
+                            Settings->WorkStationConfiguration.Farm.DefaultQueue = Value;
+                            queueChanged = true;
+                        }
+                        else
+                        {
+                            UE_LOG(LogCreateJobTest, Display, TEXT("Queue value unchanged (already '%s'), skipping update"), *Value);
+                        }
+                    }
+                }
+                else
+                {
+                    UE_LOG(LogCreateJobTest, Warning, TEXT("Failed to split pair '%s' on '='"), *KeyValuePairs[i]);
```

Should we fail here?

---

Hmmm sort of hard to know what case we'd arrive here at.  If we did we'd likely apply the same logic to a bunch of other cases as well - I tend to prefer warning over just failing for things that aren't obviously going to cause things to fail later.  I think I'd rather wait and see if this becomes a problem area, we could always start failing faster if it does